check-log/check-log.go

```diff
@@ -127,17 +146,32 @@ func run(args []string) *checkers.Checker {
 		}
 	}
 
+	msg := fmt.Sprintf("%d warnings, %d criticals for pattern /%s/.", warnNum, critNum, opts.Pattern)
+	if errorOverall != "" {
+		msg += "\n" + errorOverall
+	}
 	checkSt := checkers.OK
+	if len(missingFiles) > 0 {
+		switch opts.Missing {
+		case "OK":
+		case "WARNING":
+			checkSt = checkers.WARNING
+		case "CRITICAL":
+			checkSt = checkers.CRITICAL
+		default:
+			checkSt = checkers.UNKNOWN
+		}
+		msg += "\n" + fmt.Sprintf("%d files missing as follows.", len(missingFiles))
```

Comment:
English message is strange (?)
How about `The following %d files are missing.`.